universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java

```diff
@@ -111,16 +112,14 @@ public void stageAndUpdateIfValid_updatesMetadataForQuestionOnce() {
     ApplicantData applicantDataAfter =
         applicantRepository.lookupApplicantSync(applicant.id).get().getApplicantData();
 
-    Path programIdPath =
-        Path.create("applicant.name." + QuestionDefinition.METADATA_UPDATE_PROGRAM_ID_KEY);
-    Path timestampPath =
-        Path.create("applicant.name." + QuestionDefinition.METADATA_UPDATE_TIME_KEY);
+    Path programIdPath = Path.create("applicant.name." + Scalar.PROGRAM_UPDATED_IN.name());
+    Path timestampPath = Path.create("applicant.name." + Scalar.UPDATED_AT.name());
```

Comment:
suggestion (blocking): Might be good to `Path#join` that takes a `Scalar` to model best practices for using the `Path` API.

Reply:
done!